public/data/vishalchauhan.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Vishal Chauhan",
+  "type": "personal",
+  "bio": "OpenSource Enthusiast interested in Cloud Native Technologies",
+  "avatar": "https://avatars.githubusercontent.com/u/44545745?v=4",
```

```suggestion
  "avatar": "https://github.com/vish-han.png",
```
Please follow [To Add Your Profile](https://github.com/EddieHubCommunity/LinkFree#-to-add-your-profile) documentation. Replace the avatar with this URL format :point_right: `https://github.com/<yourusername>.png`.